Add Documentation for Energenie Pimote (#15473)
home-assistant/home-assistant.github.io: source/_integrations/energenie_pimote.markdown

---
title: "Energenie PiMote"
description: "Control your Energenie devices with a Raspberry Pi and Home Assistant"
ha_release: "0.118"
ha_category: Switch
ha_iot_class: "Assumed State"

> ha_quality_scale: ~

Reviewer: ```suggestion
```

ha_config_flow: true
ha_codeowners:
  - '@matejmecka'
ha_domain: energenie
---

> ### Energenie

Reviewer: ```suggestion
```

> This integration allows users to use their Energenie Pimote devices to be controlled via Home Assistant.

Reviewer: ```suggestion
This integration allows you to use and control your Energenie Pimote devices via Home Assistant.
```

<div class='note'>

> This integration requires the Pimote addon installed on the Raspberry pi, along with previous setup!

Reviewer: previous setup?

Author: What do you think is better way to reference it? 

`This integration requires having your switch previously assigned an ID from the link below?`

`This integration requires having followed the integration paired by giving it an ID which you can find below`

Reviewer: I'm confused by a previous setup as a whole?

What if I don't have a previous setup? Does this mean there is a prerequisite?

Author: The prerequisite would be assigning an ID to the plug the person owns, that the Energenie addon would know and communicate with

Reviewer: So, maybe I'm not understanding it or you are missing my point.

If a random new user gets here, never used anything of this before and reads: "along with previous setup" he will magically know what a previous setup is?

Author: I get it now. i'll fix it

</div>

> ### Configuration via the frontend
> 
> Make sure you have previously paired them or if you have not follow consult the following [guide](https://www.raspberrypi-spy.co.uk/2017/08/controlling-energenie-power-sockets-with-the-pi-mote-addon/).
> 
> In the settings go to `Integrations`, click on the `+` sign to add an integration and click on *Energenie*. After filling out the socket number and name the integration is ready to go!
> 
> 

Reviewer: ```suggestion
## Configuration

Make sure you have previously paired them or if you have not followed consult the following [guide](https://www.raspberrypi-spy.co.uk/2017/08/controlling-energenie-power-sockets-with-the-pi-mote-addon/).

In the settings go to `Integrations`, click on the `+` sign to add an integration and click on *Energenie*. After filling out the socket number and name the integration is ready to go!

```
